services/proctoring-service/.env

```
@@ -10,4 +10,7 @@ DB_PASSWORD=password
 DB_NAME=exam_db
 
 # AI Service URL
-AI_SERVICE_URL=http://127.0.0.1:8000/analyze_frame
\ No newline at end of file
+AI_SERVICE_URL=http://127.0.0.1:8000/analyze_frame
+
+
+JWT_SECRET=mySecretKey12345678901234567890123456789012345678901234567890
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)